lib/AST/Decl.cpp

```diff
@@ -5410,11 +5411,7 @@ bool VarDecl::isLazilyInitializedGlobal() const {
 
   // Top-level global variables in the main source file and in the REPL are not
```

Comment:
Are the above checks (hasClangNode() and isDebuggerVar()) needed at all then? The bit should not be set in those cases either.